5-AccessControl/1-call-api-roles/API/utils/guard.js

```diff
@@ -1,42 +1,37 @@
 const routeGuard = (accessMatrix) => {
     return (req, res, next) => {
-
         if (req.authInfo.roles === undefined) {
-            return res.status(403).json({error: 'No roles claim found!'});
+            return res.status(403).json({ error: 'No roles claim found!' });
         } else {
             const roles = req.authInfo['roles'];
 
             if (req.path.includes(accessMatrix.todolist.path)) {
```

Comment:
this routeGuard method is fine, but let's try to refactor it. For instance, right now it's only handling two routes (todolist and dashboard), but what if the user has more than 2? Can we make this more flexible?